sympy/sets/conditionset.py

```diff
@@ -190,12 +195,22 @@ def ok_sig(a, b):
                 ok_sig(i, j) for i, j in zip(a, b))
         if not ok_sig(self.sym, other):
             return S.false
+
+        # try doing base_cond first and return
+        # False immediately if it is False
+        base_cond = Contains(other, self.base_set)
+        if base_cond is S.false:
+            return S.false
+
+        # Substitute other into condition. This could raise e.g. for
+        # ConditionSet(x, 1/x >= 0, Reals).contains(0)
+        lmbda = Lambda((self.sym,), self.condition)
```

Comment:
We normally use "lamda" rather than "lmbda". Neither looks great though...

Reply:
I've seen some of Google's source use underscores like `lambda_`, `str_`, `int_`. Should I do that instead?

Reply:
Maybe that's better. It's not a big problem, I just mention it in passing

Reply:
I'll leave it as you suggested for now. Maybe in the future, a big PR and refractor all these kinds of problems if it becomes necessary.